src/core/crypto/singleKey.ts

```diff
@@ -264,6 +264,7 @@ export class AnySignature extends Signature {
 
   static isInstance(signature: Signature): signature is AnySignature {
     return (
+      "anySignatureVariant" in signature &&
```

Comment:
is this why you're renaming it?

Reply:
yes, I fear that the isInstance methods are not specific enough because 'data' and 'variant' are too generic.